maestral_qt/utils.py

```diff
@@ -190,8 +190,8 @@ def get_masked_image(path, size=64, overlay_text=""):
     # Crop image to a square:
     imgsize = min(image.width(), image.height())
     rect = QRect(
-        (image.width() - imgsize) / 2,
-        (image.height() - imgsize) / 2,
+        int((image.width() - imgsize) / 2),
+        int((image.height() - imgsize) / 2),
         imgsize,
```

Comment:
See previous comment.

Reply:
Actually, maybe rounding may also be more appropriate here.

Reply:
`int(round())` seems a bit messy. I liked the `//` more but I implemented like you asked.